monika.example.yml

```diff
@@ -0,0 +1,94 @@
+probes:
```

Comment:
Add a comment to point to the complete docs, e.g. https://monika.hyperjump.tech/guides/probes